tests/simulate/test_hoomd.py

```diff
@@ -0,0 +1,183 @@
+"""Unit tests for relentless.simulate.hoomd."""
+import tempfile
+import unittest
+
+import gsd.hoomd
+import hoomd
+import numpy as np
+
+import relentless
+
+class test_HOOMD(unittest.TestCase):
+    """Unit tests for relentless.HOOMD"""
+
+    def setUp(self):
+        self._tmp = tempfile.TemporaryDirectory()
+        self.directory = relentless.Directory(self._tmp.name)
+
+    #mock (NVT) ensemble and potential for testing
+    def ens_pot(self):
+        ens = relentless.Ensemble(T=100.0, V=relentless.Cube(L=10.0), N={'A':2,'B':3})
+        pot = relentless.PairMatrix(types=ens.types)
+        for pair in pot:
+            pot[pair]['r'] = np.array([1.,2.,3.])
+            pot[pair]['u'] = np.array([2.,4.,6.])
+            pot[pair]['f'] = np.array([-2.,-2.,-2.])
+        return (ens,pot)
+
+    #mock gsd file for testing
+    def create_gsd(self):
+        with gsd.hoomd.open(name=self.directory.file('test.gsd'), mode='wb') as f:
+            s = gsd.hoomd.Snapshot()
+            s.particles.N = 5
+            s.particles.types = ['A','B']
+            s.particles.typeid = [0,0,1,1,1]
+            s.particles.position = np.random.random(size=(5,3))
```

Comment:
You should probably draw these uniformly in the box. Default random numbers run [0,1), so you have a high chance of particles overlapping in the initial config. Your potential is attractive, so that's probably OK, but just something to watch out for.
```suggestion
            s.particles.position = np.random.uniform(low=-5.0,high=5.0,size=(5,3))
```